File: Calculators/Python Calculator/Nested if.py
print("Calculator in python\n")
a = int(input("Enter First Number=\n "))
b = int(input("Enter Second Number=\n "))
operation = input("Enter Operation\n + to add\n - to subtract\n * to multiply\n / to divide")

if(operation == '+'):
    {
 print(a, '+', b, '=', a+b)
    }
if(operation == '-'):
 {
print(a, '-', b, '=', a-b)
 }

if(operation == '*'):
 {
print(a, '*', b, '=', a*b)
}

if(operation == '/'):
 {
 print(a, '/', b, '=', a/b)
}

else:
  {
   print("ERROR")
 }

File: Calculators/C++ Calculator/Nested if.cpp
#include<iostream>
#include<conio.h>
  using namespace std;
int main()
   {
   cout << "Calculator\n";
   double a, b;
   char res;
   cout << "Enter first number\n";
   cin  >>  a;
   cout <<  "Enter second number\n";
   cin  >>  b;
   cout <<  "Enter Operation\n + to add\n - to subtract\n * to multiply\n \/ to divide\n";
   cin  >>  res;
   
   if(res == '+')
   {
   cout << a << "+" << b << "=" << a+b;
   }
   
   if(res == '-')
   {
   cout << a << "-" << b << "=" << a-b;
   }
   
   if(res == '*')
   {
   cout << a << "*" << b << "=" << a*b;
   }
   
   if(res == '/')
   {
   cout << a << "\/" << b << "=" << a/b;
   }
   getche();
   }
